commands/create.js

```diff
@@ -3,27 +3,39 @@ const Nuggies = require('nuggies');
 const giveaway = {};
 const ms = require('ms');
 module.exports.run = async (client, message, args) => {
-const filter = m => m.author.id === message.author.id;
-    const collector = message.channel.createMessageCollector(filter, { max: 7, time: 60 * 1000 });
-    let step = 0;
 
+    const Nuggies = require('nuggies');
+    const prettyMilliSeconds = require('pretty-ms')
```

Comment:
pretty-ms isnt required imo, the package itself parses the time into milliseconds. `endAfter` takes in type `String`  and parses it to millisecond to do the job, you can just display the raw user input in the verify part.

Reply:
It just looked nice :peepo_shrug: